Formula/curlpp.rb

```diff
@@ -15,11 +15,19 @@ class Curlpp < Formula
 
   depends_on "cmake" => :build
 
+  uses_from_macos "curl"
+
   def install
     ENV.cxx11
     system "cmake", ".", *std_cmake_args
     system "make", "install"
-    inreplace bin/"curlpp-config", "#{HOMEBREW_LIBRARY}/Homebrew/shims/mac/super/clang", "/usr/bin/clang"
```

Comment:
Can we replace this with something like:
`%r{#{HOMEBREW_SHIMS_PATH}/[^/]+/super/#{ENV.cc}}, #{ENV.cc}}`

Reply:
Thanks @iMichka, today I learned.